utils/python/tests/scripts_regression_tests.py

```diff
@@ -68,14 +43,12 @@ def test_resolve_variable_name(self):
 
     def test_CIME_unit_tests(self):
         # Find and run all the unit tests in the CIME directory tree
-        os.environ["PYTHONPATH"] = LIB_DIR
```

Comment:
This env chg had global impacts (caused pylint tests to fail on some systems). Not safe.